### ISSUE TITLE
"ros install sbcl" fails on Windows, provide alternative.

### ISSUE THREAD

**Issue by Borderliner** [ailisp/commonlisp-vscode#6]
It's very painful (and impossible, at least for me) to do "ros install sbcl" on Windows. The process fails one way or another, after trying many times. Please provide an alternative so that we can have the "Go to Definition" feature.

**Comment by ailisp** [comment]
I will do more testing on windows! I only tested Linux so far, sorry

**Comment by Borderliner** [comment]
Thank you very much. Let's keep this issue open then.

**Comment by ailisp** [comment]
Recently attempt to install roswell on my new laptop, both linux and windows failed. I created an issue in roswell repo: https://github.com/roswell/roswell/issues/443

**Comment by johnfredcee** [comment]
Yes, I was silly enough to try and install roswell on windows as well.
